src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
+const TOPIC_SEED_CONTAINER = process.env.TOPIC_SEED_CONTAINER;
+
+module.exports = {
+
+  Get: (blobName, id) => {
+    let blobSvc = azure.createBlobService(BLOB_STORAGE_CONNECTION_STRING);
```

Comment:
You should check to make sure that `blobSvc` is defined before using it, and thread the error to app insights if it's not.